core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -372,14 +382,14 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
     }
   }
 
-  def updateEntityAndWorkspace(entity: Entity, workspace: Workspace, workflowOutputs: Iterable[BoundOutputExpression]): (Option[Entity], Option[Workspace]) = {
-    val entityAttributes = workflowOutputs.collect({ case BoundOutputExpression(ThisEntityTarget, attrName, attr) => (attrName, attr) })
+  def updateEntityAndWorkspace(entity: Entity, workspace: Workspace, workflowOutputs: Iterable[BoundOutputExpression]): (WorkflowEntityUpdate, Option[Workspace]) = {
```

Comment:
This has gone from generating an `(Option[Entity], Option[Workspace])` to `(WorkflowEntityUpdate, Option[Workspace])`, which is less balanced. I could make a `WorkflowWorkspaceUpdate` class if we prefer neatness.